stac_fastapi/extensions/stac_fastapi/extensions/core/filter/filter.py

```diff
@@ -26,9 +26,9 @@ class FilterConformanceClasses(str, Enum):
     See https://github.com/radiantearth/stac-api-spec/tree/v1.0.0-rc.1/fragments/filter
     """
 
-    FILTER = "https://api.stacspec.org/v1.0.0-rc.1/item-search#filter:filter"
+    FILTER = "https://api.stacspec.org/v1.0.0-rc.1/item-search#filter"
     ITEM_SEARCH_FILTER = (
-        "https://api.stacspec.org/v1.0.0-rc.1/item-search#filter:item-search-filter"
+        "https://api.stacspec.org/v1.0.0-rc.1/ogcapi-features#filter"
```

Comment:
This should be 
`https://api.stacspec.org/v1.0.0-rc.1/item-search#filter`

https://github.com/radiantearth/stac-api-spec/tree/master/fragments/filter

Reply:
Awesome I can make that change.

Reply:
https://github.com/stac-utils/stac-fastapi/pull/417/commits/fe840fb79870c0013e14d9a8c08d01729a4c2a1c